src/app/loginComponents/accounts/external/accounts.component.html

```diff
@@ -89,8 +89,9 @@ <h3>
           (click)="link(row.source)"
           matTooltip="Link the account"
         >
-          <mat-icon>link</mat-icon>
-          <span>Link Account</span>
+          <mat-icon *ngIf="row.name !== 'ORCID'">link</mat-icon>
+          <img *ngIf="row.name === 'ORCID'" src="../../assets/images/account-logos/orcid.svg" alt="ORCID iD logo" class="orcid-id-logo" />
```

Comment:
You could alternatively do this more generically by adding a property to row in accounts.component.ts. At this point it's a wash, but if we ever have to do this for a second button, then we probably should move it into Typescript.